src/platform/silabs/wifi/SiWx/WifiInterfaceImpl.cpp

```diff
@@ -542,7 +542,7 @@ CHIP_ERROR WifiInterfaceImpl::InitWiFiStack(void)
     // Create the message queue
     sWifiEventQueue = osMessageQueueNew(kWfxQueueSize, sizeof(WiseconnectWifiInterface::WifiPlatformEvent), nullptr);
     VerifyOrReturnError(sWifiEventQueue != nullptr, CHIP_ERROR_NO_MEMORY);
-#if !defined(SL_MBEDTLS_USE_TINYCRYPT) && !(SL_MBEDTLS_USE_TINYCRYPT)
+#ifdef SL_MBEDTLS_USE_TINYCRYPT
```

Comment:
The logic here was reversed.
```suggestion
#ifndef SL_MBEDTLS_USE_TINYCRYPT
```